lite/examples/image_classification/ios/ImageClassification.xcodeproj/project.pbxproj

```diff
@@ -108,6 +108,7 @@
 			children = (
 				AAE79E58213521DA00E7AC1F /* ImageClassification */,
 				AAE79E57213521DA00E7AC1F /* Products */,
+				D48092CD801D3DF8F55ECF9A /* Pods */,
```

Comment:
The current changes in the project file seems unnecessary to me. Could you revert the changes from this file?
The Pods/ directory would be added when you do `pod install` so they don't need to be checked in either.